src/App.js

```diff
@@ -1,25 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Nav from './components/Navbar';
+import Home from './pages/Home';
+import Categories from './pages/Categories';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <>
+    <Nav />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="categories" element={<Categories />} />
```

Comment:
Kindly note that the routes should be `/categories` and not 'categories' as suggested in the project requirement , so as to make your app work better 😄

Reply:
Thank you so much. I will do that right away..